README.md

```diff
@@ -1,22 +1,22 @@
 <a id="contents"></a>
 
 # i18n-static
-Translation editable A3 pieces with i18next
+Edit translations of static text with i18next
 
 1. [Installation](#1)<br>
 2. [Configuration](#2)<br>
 3. [Behavior](#3)<br>
 4. [Import and export](#4)<br>
 
-This module adds editable pieces for translation through i18next to an A3 project.
+This module makes it possible to edit the translations of static text strings found in your templates through the ApostropheCMS admin UI.
```

Comment:
Please link to the appropriate apostrophe documentation re: static i18n so developers understand how to set this up in templates.